src/services/VerifyAccountRequestProcessor.ts

```diff
@@ -229,7 +231,7 @@ export class VerifyAccountRequestProcessor {
   			extensions:{
   				evidence:[
 						 {
-  						txn: hmrcUuid,
+  						txn: vendorUuid,
```

Comment:
Perhaps not the best place to say this - but this large extension object and restricted object looks a bit like a code smell. Should we look at if we need a builder for these objects in future?